nh_food_and_fluid/data/review_task_cron.xml

```diff
@@ -5,7 +5,7 @@
             <field name="interval_number">1</field>
             <field name="interval_type">hours</field>
             <field name="numbercall">-1</field>
-            <field eval="True" name="doall" />
+            <field eval="False" name="doall" />
```

Comment:
I understand this choice, could potentially add an extra check so that as well as checking the hour, we also see if a task is already open that was created in the same hour. Then we know if it's a repeat run. May be some problems with server being down for more than one day, but worth looking into.